bin/ncar_derecho_build.sh

```diff
@@ -7,11 +7,11 @@
 
 MODE=${1^^}
 if [ "$MODE" == "GPU" ]; then
-  sed -i 's/MODE CPU/MODE GPU/g' ../CMakeLists.txt
+  sed -i 's/MODE CPU/MODE GPU/g' CMakeLists.txt
```

Comment:
If you plan to run this script under the root directory, please update the comment at line 3 for clarification. Also do you want to update `ncar_derecho_run.sh` script as well?

Reply:
Good point, I missed the fact that it was being run in the build directory.  Being run from the main directory prevents the need to go searching for the output files, which I spent an embarrassingly large amount of time on last week.